src/main/scala/ai/privado/languageEngine/php/tagger/collection/AnnotationsCollectionTagger.scala

```diff
@@ -0,0 +1,84 @@
+package ai.privado.languageEngine.php.tagger.collection
+
+import ai.privado.cache.RuleCache
+import ai.privado.languageEngine.java.tagger.collection.CollectionUtility
+import ai.privado.model.{Constants, RuleInfo}
+import ai.privado.tagger.PrivadoParallelCpgPass
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.{Annotation, Method}
+import io.shiftleft.semanticcpg.language.*
+import org.slf4j.LoggerFactory
+
+import scala.collection.mutable
+
+class AnnotationsCollectionTagger(cpg: Cpg, ruleCache: RuleCache) extends PrivadoParallelCpgPass[RuleInfo](cpg) {
```

Comment:
The code present in this class, looks pretty much identical to other language's Annotation collection tagger. 

Create a base class with all the necessary functionality and override the differentiating piece in the language's tagger